babel/src/test/java/org/apache/calcite/test/BabelParserTest.java

```diff
@@ -327,6 +327,26 @@ private void checkParseInfixCast(String sqlType) {
     sql(sql).ok(expected);
   }
 
+  @Test public void testTableAttributeJournalTable() {
+    final String sql = "create table foo, with journal table = baz (bar integer)";
+    final String expected = "CREATE TABLE `FOO`, WITH JOURNAL TABLE = `BAZ` (`BAR` INTEGER)";
+    sql(sql).ok(expected);
+  }
+
+  @Test public void testTableAttributeJournalTableWithSource() {
+    final String sql = "create table foo, with journal table = baz.tbl (bar integer)";
+    final String expected = "CREATE TABLE `FOO`, WITH JOURNAL TABLE = `BAZ`.`TBL` (`BAR` INTEGER)";
+    sql(sql).ok(expected);
+  }
+
+  @Test public void testTableAttributeMultipleAttrs() {
```

Comment:
Should we also test that the order of attributes doesn't matter?

Reply:
That's a good point, I can add another test for that too

Reply:
Good catch, thanks!